app-backend/common/src/main/scala/CommonHandlers.scala

```diff
@@ -0,0 +1,14 @@
+package com.azavea.rf.common
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.StandardRoute
+import akka.http.scaladsl.server.directives.RouteDirectives
+
+
+trait CommonHandlers extends RouteDirectives {
```

Comment:
I see what you mean, and I think I like keeping them distinct. I think it might be useful to have a `CommonErrorHandlers` trait if we ever come up with more than `UserErrorHandlers`, but I don't think they should be combined with `CommonHandlers` because of the success / exception result.

I'm good with the redundancy for now, and we can think of combining them if they evolve in that direction down the line.